CactbotOverlay/CactbotEventSource.cs

```diff
@@ -157,58 +194,54 @@ public CactbotOverlay(CactbotOverlayConfig config)
       fast_update_timer_.Interval = kFastTimerMilli;
       fast_update_timer_.Start();
 
-      if (is_first_overlay_initialized_) {
```

Comment:
Does this mean that every cactbot overlay will print this out? Is there a reason this doesn't work with your new approach?

Reply:
`Start` is only called once when OverlayPlugin loads the addon. Thus every call to `Start` would be the "first overlay" making this check redundant.

Reply:
Oh, I get it.  Thanks.